packages/shared/src/styles/base.css

```diff
@@ -87,6 +87,7 @@ body {
   --theme-shadow2: theme('boxShadow.2-black');
   --theme-shadow3: theme('boxShadow.3-black');
 
+  --theme-blur-highlight: rgba(27, 31, 38, 0.88);
```

Comment:
I think we already have this blur style somewhere.
Is it actually not the same as the video blur?

If not @capJavert also recently introduced one, might be the same hit.

If not the same, what color/setup is this from our guideline, might have an alternative idea.

Reply:
> Is it actually not the same as the video blur?

No, it is much darker. This feels like something new from the guideline they call blur. I also remember introducing some blur in the past, can't exactly find where it is now 😅

Reply:
![image](https://github.com/dailydotdev/apps/assets/13744167/32d9f15b-86be-4eab-8360-965614eec8ac)

Reply:
Ah yes I think this is exactly the one Ante added already then.
Let's see if we can reuse that rather else we need to check the actual values. We should not add raw hex colors for now. (its probably just a figma issue you can't see the named value)

Reply:
The reason we had to go for the raw hex colors is, that the color used here is not found in the guideline. It

Reply:
Yes, but did you check it with design team? 🙏
It's often like 1 number off because of a glitch.

I think he introduced it on search merge?
https://github.com/dailydotdev/apps/pull/2510/files#diff-501943f96c58a477247d24220eb9c8fca482b85b5e60de358ad8e1500026ee8eR124

Not a 100% sure if it's the same, but let's see.

Reply:
Agree here, let's try to not introduce new things unless design specifically mentions. If we can't find the colors/blur, let's pick the most similar ones we have. We can leave a note for this to design, so that they are aware and they can let us know if we need to change for a reason.

Reply:
> Agree here, let's try to not introduce new things unless design specifically mentions.

When it is labeled as shared on the screenshot, that usually means it is part of our design system.

> Not a 100% sure if it's the same, but let's see.

After checking, the value used was related to `backdrop-blur` - which is a totally different setting.